### DISCUSSION TITLE
关系代数的基本运算

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#5] [Announcements]
# 关系代数的基本运算

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/2017/02/19/basic-operations-of-relation-algebra

**Comment by liuhedev** [comment]
差集部分，写错了吧？应该是R−S={t|t∈R∧t∉S}

**Reply by keelii** [reply]
感谢提醒，已更正。

**Comment by Jiale-Li** [comment]
真棒，终于发现一个介绍这么完整的宝藏

**Comment by njuhesiyuan** [comment]
很有用的总结，对我的学习帮助很大
